step-framework-parent/step-framework-timeseries/src/main/java/step/core/timeseries/metric/MetricTypeRenderSettings.java

```diff
@@ -0,0 +1,51 @@
+package step.core.timeseries.metric;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * This class contains all needed settings for rendering a visual chart.
+ */
+public class MetricTypeRenderSettings {
+    
+    private String groupingAttribute; // the default grouping
```

Comment:
Does this correspond to the list of attributes supported by this metric for grouping?

Reply:
@jeromecomte I don't exactly get the question. This is the grouping dimension. e.g: "name", or "status"

Reply:
OK, I understand. For which metric do we need a default grouping attribute?